### ISSUE TITLE
No parser configuration found for subdirectory cspTrustedSites

### ISSUE THREAD

**Issue by alexanderyhapcv** [SFDO-Tooling/CumulusCI#898]
Error when running `cci flow run deploy_unmanaged`

Is the cspTrustedSites metadata type supported in CCI?

```
  File "/Users/alexander/Google Drive/ayhap@causeview/cvGitHub/CV-Advancement-CCI/cci/lib/python3.7/site-packages/cumulusci/tasks/metadata/package.py", line 85, in parse_types
    "No parser configuration found for subdirectory %s" % item
cumulusci.tasks.metadata.package.MetadataParserMissingError: No parser configuration found for subdirectory cspTrustedSites
```

**Comment by jlantz** [comment]
I don't think we have a mapping for that metadata type.  We don't aim to support every metadata type at this point as doing so would require a lot of ongoing work.  Fortunately, adding support for a metadata type is pretty easy.  This yaml file controls how our package.xml generation logic handles metadata types in the local directory:

https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/tasks/metadata/metadata_map.yml#L31

Checking out the docs for this type, it looks like it's just a single file in a named directory for each instance:
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_csptrustedsite.htm

In metadata_map.yml that could be handled pretty easily with the `MetadataFilenameParser`
